WhaleSpotting.UnitTests/WhaleSpotting.UnitTests.csproj

```diff
@@ -5,9 +5,15 @@
         <IsPackable>false</IsPackable>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Compile Remove="Controllers\GetInfoTest.cs" />
+      <Compile Remove="Services\SightingsServiceTest.cs" />
+    </ItemGroup>
```

Comment:
We don't need these so they can be removed :)

Reply:
Fixed